### ISSUE TITLE
low cpu usage

### ISSUE THREAD

**Issue by whoamikyo** [FallingSnow/h265ize#137]
have a epyc cpu with 32 cores and the cpu usage is very low, how can i fix that?

![](https://i.imgur.com/PCgghp2.png)

**Comment by FallingSnow** [comment]
Probably just a limit of the h265 encoder implementation used by ffmpeg (libx265).

Nothing to do that is really desirable. You could probably tweak some x265 settings to make it use more CPU, the output would come with a better perceptual quality to file size ratio, but I can almost guarantee that it would encode even slower.

This is due to work that is serial and cannot be effectively paralleled, so the rest of the threads need to wait a little before continuing (aka the cpu usage drop).

tl;dr Nothing's wrong, that just as fast as ffmpeg can encode that video.

**Comment by FallingSnow** [comment]
Best solution to make use of your time and extra CPU is just to do another encode in parallel (run h265ize on another video file at the same time).

**Comment by Appyx** [comment]
Would it be possible to add a parameter which lets the user choose the number of parallel encodings when encoding a folder?

**Comment by FallingSnow** [comment]
Unfortunately I believe this is something that should be done outside of h265ize since h265ize has kind of "unofficially" reached EOL.

**Comment by timonline** [comment]
> Unfortunately I believe this is something that should be done outside of h265ize since h265ize has kind of "unofficially" reached EOL.

If I may ask why is it EOL and is there a better alternative?
